cachix-api/ChangeLog.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.1.0.2] - 2018-09-27
+
+### Changed
+
+- #133 Address all GHC warnings @domenkozar
+- #132 Remove autogenerated cabal file from git @domenkozar
+- NarC now strips the extension @domenkozar
+- added `BinaryCacheAuthenticated` @domenkozar
+- #118 User: add email field @domenkozar
+- Stackage LTS-12 @domenkozar
```

Comment:
nitpick: Update dependencies to Stackage LTS-12

In case someone is not familiar with the Haskell ecosystem.